### DISCUSSION TITLE
Tag autocomplete by most commonly used tags first when creating/editing articles

### DISCUSSION THREAD

**Discussion by cirosantilli** [realworld-apps/realworld#784] [Ideas]
It would be cool to illustrate tag autocomplete, as you start typing `abc` it shows a dropdown of all existing tags `abcXXX` with most commonly used first. On the backend we could maybe have `/api/tags?startWith=XXX`, and then everything else is frontend.

**Comment by geromegrignon** [comment]
Agreed 👍
